ui/app/src/components/PreviewSearchPanel/PreviewSearchPanel.tsx

```diff
@@ -137,13 +145,17 @@ export function PreviewSearchPanel() {
 
   const onSearch = useCallback(
     (keywords: string = '', options?: ComponentsContentTypeParams) => {
+      let sub;
       setState({ isFetching: true });
       setError(null);
-      search(site, {
+      const allowedTypes = Object.entries(allowedTypesData ?? {}).flatMap(([key, type]) => (type.shared ? [key] : []));
+
+      sub?.unsubscribe();
```

Comment:
This will always be undefined here. Again, don't see why not declare it inline in the next line.